### PR TITLE
Minor readability patch.

### PR DESCRIPTION
**Pull request by mikayelgr**
Please refer to [this conversation](https://discord.com/channels/322538954119184384/322910365237248000/908433728571863130) on Gitea's Discord server for more info.

### CODE REVIEW

**Comment by techknowlogick** [comment]
CI fails because of gofmt warnings. To remedy please run make fmt

**Comment by michaelgrigoryan25** [comment]
I see why. I think I've made a typo.

**Comment by michaelgrigoryan25** [comment]
Any ideas why it keeps failing?

**Comment by qwerty287** [comment]
> CI fails because of gofmt warnings. To remedy please run make fmt

See the comment by @techknowlogick - you have to run `make fmt` and commit and push the changes made by this command to fix it

**Comment by michaelgrigoryan25** [comment]
I did actually but ended up getting an error, with nothing except the error code.

File: services/auth/basic.go [hunk 1]
```diff
@@ -50,8 +50,8 @@ func (b *Basic) Verify(req *http.Request, w http.ResponseWriter, store DataStore
 	}
 
 	auths := strings.SplitN(baHead, " ", 2)
-	if len(auths) != 2 || (auths[0] != "Basic" && auths[0] != "basic") {
-		return nil
+	if len(auths) != 2 || (strings.ToLower(auths[0]) != "basic") {
+    	return nil
```
Comment (techknowlogick) [inline services/auth/basic.go]:
```suggestion
 		return nil
```

File: services/auth/basic.go [hunk 2]
```diff
@@ -50,8 +50,8 @@ func (b *Basic) Verify(req *http.Request, w http.ResponseWriter, store DataStore
 	}
 
 	auths := strings.SplitN(baHead, " ", 2)
-	if len(auths) != 2 || (auths[0] != "Basic" && auths[0] != "basic") {
-		return nil
+	if len(auths) != 2 || (strings.ToLower(auths[0]) != "basic") {
+ 		return nil
```
Comment (techknowlogick) [inline services/auth/basic.go]:
```suggestion
		return nil
```

**Comment by techknowlogick** [comment]
@michaelgrigoryan25 hope you don't mind, but I pushed the lint placate to your branch. It now passes the lint step of CI

**Review by lafriks — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Comment by michaelgrigoryan25** [comment]
Thanks for your help! And just to clarify, was it failing because of spacing?

**Comment by techknowlogick** [comment]
> Thanks for your help! And just to clarify, was it failing because of spacing?

That's correct. Golang has strong opinions on formatting and so one of our linters checks that each PR adheres to that opinion to ensure maximum compatibility between PRs from many different contributors

**Pull request merged**